querydsl-apt/src/main/java/com/querydsl/apt/NamingFunction.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2016, The Querydsl Team (http://www.querydsl.com/team)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.querydsl.apt;
+
+import com.google.common.base.Function;
+import com.querydsl.codegen.EntityType;
+
+/**
+ * The {@link Function} signature to convert {@link EntityType} to {@link String}.
+ *
+ * @author Shredder121
+ */
+public interface NamingFunction extends Function<EntityType, String> {
```

Comment:
That's up to you.
We'll probably have to add a new `catch` for `CCE` and a friendly notice that we moved to a better interface :wink: